DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -462,6 +462,11 @@ def run_traffic_item(self, device, command, *argv, **kwarg):
         elif command == "clear_stats":
             device.applog.info("Clear Stats")
             IxnetworkIxiaClientImpl.ixnet.ClearStats()
+        elif command == "clear_traffic":
+            device.applog.info("Clear Traffic")
```

Comment:
Please specify in this message which TIs are about to be removed.
And since the previous log message is "Clear Traffic Items on ...", this message could be shortened to "TIs to be removed: ...."

Reply:
"Clearing traffic items..."